lib/app/4_view/2_home/home_binding.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:cnubot_app/app/1_data/1_provider/shuttle_provider.dart';
+import 'package:cnubot_app/app/1_data/2_repository/shuttle_repository.dart';
+import 'package:cnubot_app/app/2_dio/dio_helper.dart';
+import 'package:get/get.dart';
+
+import 'home_controller.dart';
+
+class HomeBinding implements Bindings {
+  @override
+  void dependencies() {
+    Get.lazyPut<HomeController>(() {
+      return HomeController(
+        repository: ShuttleRepository(
+          provider: ShuttleProvider(
+            dioHelper: DioHelper(),
+          ),
+        ),
+      );
+    });
```

Comment:
DI 대박 깔끔하게 해주셨네요...
일반적으로 Get.put 보다는 lazyPut 을 사용하시는 편이신가요?
특별한 이유가 있으신가요?!

Reply:
- `It is possible to lazyLoad a dependency so that it will be instantiated only when is used. Very useful for computational expensive classes or if you want to instantiate several classes in just one place (like in a Bindings class) and you know you will not gonna use that class at that time.`
- `Get.put() will put immediately`
- `Get.lazyPut() will put when you need it`

해당 클래스가 사용되기 직전에만 때만 인스턴스화 돼서 컴퓨팅 비용적으로 효율적이여서 좋다고 합니다!